Sources/Bag.swift

```diff
@@ -48,14 +48,12 @@ public struct Bag<Element> {
 	///   - token: A token returned from a call to `insert()`.
 	@discardableResult
 	public mutating func remove(using token: Token) -> Element? {
-		for i in elements.indices.reversed() {
-			if tokens[i] == token.value {
-				tokens.remove(at: i)
-				return elements.remove(at: i)
-			}
+		guard let i = indices.first(where: { tokens[$0] == token.value }) else {
```

Comment:
[Nit] Could you replace `i` with `index`?

Reply:
👍